msyncd/AccountsHelper.cpp

```diff
@@ -100,7 +100,6 @@ void AccountsHelper::slotAccountCreated(Accounts::AccountId id)
                 delete syncProfile;
             }
         }
-        delete newAccount;
     }
```

Comment:
If you prefer, I can change the function to use the fromId() method instead - that will ensure minimal memory usage.